docs/testing/runtimes.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Runtimes in Office Add-ins
+description: Learn about the JavaScript runtimes that are used by Office Add-ins.
+ms.date: 08/10/2022
+ms.localizationpriority: medium
+---
+
+# Runtimes in Office Add-ins
+
+Office add-ins execute in JavaScript runtimes embedded in Office. As an interpreted language, JavaScript must run in a JavaScript engine. As a single-threaded, synchronous language, JavaScript has no inherent capacity for concurrent execution; but modern JavaScript engines can request concurrent operations (including networking communication) from the host operating system and receive data from the OS in response. This kind of engine makes JavaScript *effectively* asynchronous. In this article, engines of this sort are called *runtimes*. [Node.js](https://nodejs.org) and modern browsers are examples of such runtimes. 
+
+## Types of runtimes
+
+There are two types of runtimes used by Office Add-ins:
+
+- **JavaScript-only runtime**: A JavaScript engine supplemented with support for [WebSockets](https://developer.mozilla.org/docs/Web/API/WebSockets_API), [Full CORS (Cross-Origin Resource Sharing)](https://developer.mozilla.org/docs/Web/HTTP/CORS), and [local storage](https://developer.mozilla.org/docs/Web/API/Window/localStorage). 
+- **Browser runtime**: Includes all the features of a JavaScript-only runtime and adds a [rendering engine](https://developer.mozilla.org/docs/Glossary/Rendering_engine) that renders HTML.
+
+Details about these types are later in this article at [JavaScript-only runtime](#javascript-only-runtime) and [Browser runtime](#browser-runtime).
+
+The following table shows which possible features of an add-in uses each type of runtime. 
+
+> [!NOTE]
```

Comment:
I think that would be going too far. Raising its importance would cause alarm and worry for the reader. If it were up to me, we wouldn't have this note at all because there really isn't anything that the developer **could** do that makes the kind of assumption that this note warns against. This note is only present to please an SME.

Reply:
OK by me